root/documents/README.md

```diff
@@ -12,7 +12,11 @@ Personal Documents like Letters, Poems, Forms, Bills, etc.
 
 ## Patterns
 
+The most common file naming scheme for documents is likely the use of the date in ISO8601 format at the start of the name (i.e YYYY-MM-DD) followed by a dash and then a useful name for the file. These names should be appropriately descriptive and formatted in an easily readable manner that allow it to be distinguished from other documents in the same subsection.
```

Comment:
Well #10 would like to disagree. I agree that the ISO pattern is very common. I'd also like to keep it as a good example. But I'm not sure if it's the most common one. Also would love to see other patterns here. Probably part of a later commit..

Reply:
I'll change that to "one of the most common", then, generally it's what I see as the most optimal, but there's definitely other options which should be included (like #10)